### ISSUE TITLE
Feature request: photo slider inside albums

### ISSUE THREAD

**Issue by LNT00** [pandy06269/blue-twilight#66]
I didn't find a photo previewer other than slideshow, and when i open a photo from an Album it opens the photo in full screen.

I would like to have a slider where i can go through all pictures in an album, without using the slideshow function. Similar to this:
http://www.foliopages.com/demos/folio-gallery/demo.php

http://www.foliopages.com/demos/folio-gallery/demo.php?album=Big+Bear+California

![previewer](https://user-images.githubusercontent.com/32333032/31124493-5becefde-a84d-11e7-9acc-daa8f56fcd25.jpg)

**Comment by pandy06269** [comment]
This sounds good. Possibly another “view” called Lightbox that displays similar to the default but opens photos in a lightbox effect instead of a new page.

It could update the view behind it so as you’re scrolling through in the lightbox, it could make sure it rotates the pages correctly.

What do you think?

**Comment by pandy06269** [comment]
By the way you do have next/previous buttons in the default view now so you could scroll through that way too, although it does cause a full page reload each time.

**Comment by LNT00** [comment]
Yes, Lightbox is what i'm thinking, just couldn't remember the name.

I think i saw an issue there when using Next/Previous, i think it's showing them random in both cases. I will look again to make sure.
